doc/nrf/known_issues.rst

```diff
@@ -357,6 +357,14 @@ Bluetooth LE
 
 .. rst-class:: v1-5-0 v1-4-2 v1-4-1 v1-4-0 v1-3-2 v1-3-1 v1-3-0 v1-2-1 v1-2-0 v1-1-0 v1-0-0
 
+NCSDK-8321: NUS shell transport sample doesn't display the initial shell prompt *uart:~$* on the remote terminal.
+  Also few logs with sending errors are displayed on the terminal connected directly to the DK.
+  This issue is caused by the shell enabling before turning on the notification for the NUS service by the remote peer.
```

Comment:
```suggestion
  This issue is caused by the shell being enabled before turning on the notifications for the NUS service by the remote peer.
```

Reply:
Done

Reply:
One more minor thing: We should avoid using contractions (isn't, doesn't etc) in the docs. This is not very bad, but maybe we should have does not instead of doesn't.

Reply:
Ok. I changed to the full form